packages/mobile/screens/sign_in.tsx

```diff
@@ -0,0 +1,196 @@
+import { useMutation } from "@apollo/react-hooks";
+import { Field , Formik } from 'formik';
+import * as React from "react";
+import { Image, StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import * as Yup from "yup";
+import gql from "graphql-tag";
+// import {LOG_IN_MUTATION} from "@workspace-library/core";
+
+const LOG_IN_MUTATION = gql`
+  mutation login($email: String!, $password: String!) {
+    logInUser(userInput: { email: $email, password: $password }) {
+      id
+    }
+  }
+`;
+
+const LoginSchema = Yup.object().shape({
+    email: Yup.string()
+        .email("Please enter a valid email")
+        .required("Please enter an email"),
+    password: Yup.string().required("Required")
+});
+
+
+export default function SignUp({ navigation }: { navigation: any }) {
+    const [logInUser, { data }] = useMutation(LOG_IN_MUTATION);
+
+    const submition = (values: any, { setErrors }: any) => {
+        logInUser({
+            variables: {
+                email: values.email,
+                password: values.password
+            }
+        })
+            .then(
+                ({
+                    data: {
+                        logInUser: { id }
+                    }
+                }) => {
+                    navigation.navigate("home")
+                }
+            )
+            .catch(({ graphQLErrors }) => {
+                const error = graphQLErrors?.map((err: any) => err?.message);
+                setErrors({ server: error[0] });
+            });
+    };
+
+    return (
+        <View style={styles.container}>
+            <View>
+                <Image resizeMode={'cover'}
+                    source={require('../assets/logo-lithium.png')}
+                />
+                <Text style={styles.welcomeText} >{'Welcome to Lithium KB'}</Text>
```

Comment:
We're trying to use [styled-components](https://styled-components.com/docs/basics)!, please change your components for a styled one.